_posts/2025-04-03-geoserver-2-27-0-released.md

```diff
@@ -230,7 +236,7 @@ A file system sandbox is used to limit access for GeoServer Administrators and W
 
 * A system sandbox is established using ``GEOSERVER_FILESYSTEM_SANDBOX`` application property, and applies to the entire application, limiting GeoServer administrators to the ``<sandbox>`` folder, and individual workspace administrators into isolated ``<sandbox>/<workspace>`` folders.
 
-* A regular sandbox can be configured from the **Security > Data** screen, and is used to limit individual workspace administrators into ``<sandbox>/<workspace>`` folders to avoid accessing each others files.
+* A regular sandbox can be configured from the **Security > Data** screen, and is used to limit individual workspace administrators into ``<sandbox>/<workspace>`` folders to avoid accessing each other's files.
```

Comment:
This was in a prior blog post also due to back port. So you may wish to update their also.